docs/pages/build/android-builds.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Android Builds
+---
+
+This page describes the process of building Android projects with EAS Builds. Dive in if you're interested in the implementation details of our build service.
+
+## Build Process
+
+Let's take a closer look at the steps for building Android projects with EAS Builds.
+
+### Local Steps
+
+The first phase is happening on your computer, Expo CLI is in charge of completing the following steps:
+
+1. Check if the git index is clean - this means if there aren't any uncommitted changes. If it's not clean an error is thrown.
+2. Prepare the credentials needed for the build unless `builds.android.PROFILE_NAME.withoutCredentials` is set to `true`.
```

Comment:
do we support that? I think project configuration on iOS required credentials

Reply:
(discussed offline) This page is for Android, not for iOS.